modulemd/v2/include/modulemd-2.0/modulemd-module-stream.h

```diff
@@ -78,6 +78,8 @@ struct _ModulemdModuleStreamClass
                                        const gchar *module_name,
                                        const gchar *stream_name);
 
+  gboolean (*equals) (ModulemdModuleStream *self, ModulemdModuleStream *other);
+
   /* Padding to allow adding up to 8 new virtual functions without
    * breaking ABI. */
   gpointer padding[8];
```

Comment:
When adding a new virtual function, you need to decrement the padding size. The reason for this is so that we don't break ABI compatibility; the size of the struct doesn't change. If you just add a new function without reducing this padding value, the struct would increase in size by one pointer and any application compiled against the previous version would be forced to rebuild to understand the new size.

The padding is there to avoid the rebuild when adding new functions.